openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/TestSchemas.java

```diff
@@ -105,14 +107,14 @@ public class TestSchemas {
      *
      * Raw data columns @ test_data/in_train_val.csv: "card","amount","event_timestamp","is_fraud_label",
      * "uuid","cat1_generator","cat2_generator","cat3_generator","num1_float","num2_double","num3_int",
-     * "cat1_string","cat2_string","cat3_string"
+     * "cat1_string","cat2_string","cat3_string", "sensitive_group"
      */
     public static final DatasetSchema CATEGORICALS_SCHEMA_LABEL_AT_END = new DatasetSchema(
-            7,
+            8,
             ImmutableList.<FieldSchema>builder()
                     .addAll(CATEGORICAL_SCHEMA_FEATURES)
-                    .add(new FieldSchema("is_fraud_label", 7,
-                            new CategoricalValueSchema(true, ImmutableSet.of("FALSE", "TRUE"))))
+                    .add(new FieldSchema("is_fraud_label", 8,
+                            new CategoricalValueSchema(true, ImmutableSet.of("False", "True"))))
```

Comment:
Why did we change this? We changed the label format on disk?

Reply:
yes, I added a new "sensitive_attribute" column using a few lines of Python and the True/False values changed after saving the new dataframe :sweat_smile: